lms/djangoapps/course_blocks/transformers/helpers.py

```diff
@@ -0,0 +1,38 @@
+"""
+Transformers helpers functions.
+"""
+from openedx.core.djangoapps.user_api.partition_schemes import RandomUserPartitionScheme
+from openedx.core.djangoapps.course_groups.partition_scheme import CohortPartitionScheme
+# TODO 8874: Make it so we support all schemes instead of manually declaring them here.
+INCLUDE_SCHEMES = [CohortPartitionScheme, RandomUserPartitionScheme, ]
+SCHEME_SUPPORTS_ASSIGNMENT = [RandomUserPartitionScheme, ]
+
+
+def get_user_partition_groups(course_key, user_partitions, user):
```

Comment:
How is this different from `_get_user_partition_groups` defined in UserPartitionTransformer.py?  Should the latter be replaced by this?